spec/requests/onboarding/signal_spec.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Onboarding::Signals', type: :request do
```

Comment:
```suggestion
RSpec.describe 'Onboarding::Signal', type: :request do
```

Reply:
Why do the tests run through with both versions?

Reply:
I think as long as you don’t use [`described_class`](https://relishapp.com/rspec/rspec-core/docs/metadata/described-class) somewhere in your tests, what you pass in to `describe` has no effect on your tests (except that it’ll be used in the console output when running the tests).

Reply:
it changes the `subject`